Source/CarthageKit/Project.swift

```diff
@@ -609,6 +605,54 @@ private func filesInDirectory(directoryURL: NSURL, typeIdentifier: String) -> Si
 		}
 }
 
+/// Sends the URL for the Info.plist of the specified Framework.
+private func infoPlistForFramework(frameworkURL: NSURL) -> SignalProducer<NSURL, CarthageError> {
+	return filesInDirectory(frameworkURL, "com.apple.property-list")
+		|> filter { plist in plist.lastPathComponent == "Info.plist" }
```

Comment:
@neilpa To clarify: iOS frameworks do not appear to have the `Resources` directory by default, so I'm not sure that this will work in all cases. That document appears to only pertain to Mac OS X frameworks.